hypothesis-python/src/hypothesis/internal/validation.py

```diff
@@ -85,6 +85,7 @@ def check_valid_magnitude(value, name):
             "min_magnitude=None is deprecated; use min_magnitude=0 "
             "or omit the argument entirely.",
             since="2020-05-13",
+            has_codemod=True,
```

Comment:
If it is still possible, I would try to use something like `codemod=HypothesisFixPositionalKeywonlyArgs` instead of `has_codemod=True`.

In my view it is better to make this link phisical. It would allow to have better docs and better help in the future.

But, I guess there might be import issues 🤔

Reply:
> But, I guess there might be import issues 🤔

Yep - because `libcst` is an optional dependency we can't reliably pass the codemod object.  We could pass the codemod *name* though, if that would help.  How would you use that to improve the warning message?

I just went for a boolean because I'd recommend the simple `hypothesis codemod` CLI for most users and that doesn't support selecting individual codemods.

Reply:
> codemod=HypothesisFixPositionalKeywonlyArgs

Ok, I see 🙂  
This would make perfect sense when we would have several codemods. 
In this case, it would be easier: you have an old version of `X`? Use codemod `A`, run `hypothesis codemod A`.

Because running all codemods at once might not be desireable. (That's something I learned the hard way from writting several code-quality tools).

Reply:
> Because running all codemods at once might not be desireable. (That's something I learned the hard way from writting several code-quality tools).

Agreed - but you can get that via e.g. `python -m libcst.tool codemod codemods.HypothesisFixPositionalKeywonlyArgs`.

It's a little more verbose, but because the full-control interface is also available I've kept `hypothesis codemod` *ruthlessly* simple, so that you don't pay the cost of having config options unless you opt-in.

Reply:
Great! I was not aware of this mode (and tool)! 👍 
Thanks for showing it 🙂